custom/conf/app.example.ini

```diff
@@ -682,6 +682,28 @@ LEVEL = Info
 ;; Disable the usage of using partial clones for git.
 ;DISABLE_PARTIAL_CLONE = false
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Git Operation timeout in seconds
+;[git.timeout]
+;DEFAULT = 360
+;MIGRATE = 600
+;MIRROR = 300
+;CLONE = 300
+;PULL = 300
+;GC = 60
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Git Reflog timeout in days
+;[git.reflog]
+;ENABLED = true
+;EXPIRATION = 90
```

Comment:
I think we can already put it inside `[git.config]`.
If I remember that PR correctly, it did nothing else than forward these settings to the git config.

Reply:
The only real change in that PR was that it enables the reflog by default instead of disabling it.

Reply:
Yup, it's done by #22451 , but it had been backported .... so we can't just drop the `[git.reflog]`, it's another breaking ....

Reply:
But that's exactly what I mean.
Otherwise this PR isn't breaking.

Reply:
So I think we can keep this PR as non-breaking, and "break" the `[git.reflog]` in a separate PR

Reply:
Pre-1.20.0-rc0?

Reply:
Do you mean to "break" `[git.reflog]` in 1.20 release? If you agree, I can do it in the following PR.

Reply:
Yes, that's what I meant.
I want to get rid of that technical debt as fast as possible.